src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month month--month-length--30">
+      <div class="month__day month__day--start-day--wed"></div>
```

Comment:
both modifiers should be for a block month, not for the first child element

```suggestion
    <div class="month month--month-length--30 month__day--start-day--wed">
      <div class="month__day"></div>
```